src/main/java/frc/robot/commands/AutoScoreAndBalance.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Calibrations;
+import frc.robot.subsystems.Lift;
+import frc.robot.subsystems.drive.DriveSubsystem;
+
+public class AutoScoreAndBalance extends SequentialCommandGroup {
```

Comment:
Let's add a chooser to RobotContainer so we can choose which auto to run. See here: https://docs.wpilib.org/en/stable/docs/software/dashboards/smartdashboard/choosing-an-autonomous-program-from-smartdashboard.html